### DISCUSSION TITLE
Perhaps adding Client Identificaiton to the email sent by Remotely?

### DISCUSSION THREAD

**Discussion by futuredata123** [immense/Remotely#311] [Ideas]
My thought is this; if the client information such as **machine name**, and **group** were added to the email sent out by Remotely that would be incredibly useful! 

Say in the case of multiple organizations that find their home on one server, the email providing the organization the machine is subscribed to could also be quite useful. 

Although I am not sure how complicated it would be to integrate.

**Comment by bitbound** [comment]
I've added the device ID, device name (machine name), device alias, and group name (if any) to the email.  :)

It will be included in the next release.

**Reply by futuredata123** [reply]
That's AWESOME! I'm so stoked to show that off! Thank you again for your incredible hard work!
